### PR TITLE
support audio buffer node detune

### PR DESCRIPTION
**Pull request by ngokevin**
The last audio buffer source node property to support, next to playbackRate.

https://developer.mozilla.org/en-US/docs/Web/API/AudioBufferSourceNode/detune

### CODE REVIEW

File: docs/api/en/audio/Audio.html [hunk 1]
```diff
@@ -73,6 +73,9 @@ <h3>[property:Boolean hasPlaybackControl]</h3>
 		<p>Whether playback can be controlled using the [page:Audio.play play](),
 			[page:Audio.pause pause]() etc. methods. Default is *true*.</p>
 
+		<h3>[property:Number detune]</h3>
+		<p>Modify pitch. Default is *0*.</p>
```
Comment (Mugen87) [inline docs/api/en/audio/Audio.html]: I think I would add the metric which is `cent`.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by Mugen87** [comment]
@mrdoob Should we provide `.getDetune()` and `.setDetune()` for consistency reasons?

**Comment by mrdoob** [comment]
@ngokevin could you address @Mugen87's comments?

**Comment by ngokevin** [comment]
Updated.

File: src/audio/Audio.js [hunk 2]
```diff
@@ -260,6 +262,26 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 	},
 
+	setDetune: function ( value ) {
+
+		this.detune = value;
+
+		if ( this.isPlaying === true ) {
+
+			this.source.detune.setValueAtTime( this.detune, this.context.currentTime );
```
Comment (Mugen87) [inline src/audio/Audio.js]: Is there a special reason for using `setValueAtTime()`? We normally use `setTargetAtTime()`, see https://github.com/mrdoob/three.js/pull/11133#issuecomment-400815949

```js
this.source.detune.setTargetAtTime( this.detune, this.context.currentTime, 0.01 );
```

Comment (Mugen87) [reply src/audio/Audio.js]: Please also add both new methods to the documentation.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

File: src/audio/Audio.js (hunk 2 above)
Comment (ngokevin) [reply src/audio/Audio.js]: I copied the playbackRate code, but I can change both.

Comment (mrdoob) [reply src/audio/Audio.js]: Yeah, that'd be great 😊

**Pull request closed without merging**